### PR TITLE
DOC-645: Restructure menu item and toolbar button refs

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-645

Description of Changes:
* Updated links to the toolbar button and menu item identifiers.
* Restructured the toolbar button and menu item page into two separate pages.
* Split the toolbar button and menu item tables into core and individual plugins so the content could be reused on plugin pages.
* Added the relevant menu items and toolbar buttons to the related plugin pages.
* Changed the name of some pages in the advanced section of the docs
* Added some reusable text chunks required for the task.
* Moved the "Tiny Drive" plugin from `/plugins/drive/` to `/plugins/tinydrive/`.

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _data/nav.yml [hunk 1]
```diff
@@ -680,12 +680,99 @@
...
   - url: "editor-command-identifiers"
     pages:
-      - url: "#Executable commands"
-      - url: "#Query command states"
-      - url: "#Query command values"
-  - url: "editor-control-identifiers"
+    - url: "#Executable commands"
+    - url: "#Query command states"
+    - url: "#Query command values"
+  #- url: "editor-control-identifiers"
```
Comment (tylerkelly13) [inline _data/nav.yml:775]:
```suggestion
```

File: plugins/powerpaste.md [hunk 2]
```diff
@@ -169,21 +172,11 @@ tinymce.init({
...
-tinymce.init({
-  selector: 'textarea',
-  plugins: 'powerpaste',
-  toolbar: 'pastetext'
-});
-```
+{% assign altplugincode = "paste" %}
+{% include misc/plugin-menu-item-id-boilerplate.md %}
 
 ## Menu Items
```
Comment (lnewson) [inline plugins/powerpaste.md:181]: Menu items is duplicated here

File: _data/nav.yml [hunk 3]
```diff
@@ -268,7 +268,6 @@
   - url: "comments"
```
Comment (lnewson) [inline _data/nav.yml:268]: We've added new sections to all the plugin pages (`Menu Items` and `Toolbar Buttons`). Should they be included in the nav here?

We include `Commands` and `API` in most cases, so I think we should include these new sections.

**Review by lnewson — APPROVED** [review]
Overall LGTM 👍  Just left a couple things to consider though

File: plugins/powerpaste.md (hunk 2 above)
Comment (tylerkelly13) [reply plugins/powerpaste.md]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1675/commits/9189463c6115d08e4d00e072ea557288f9416116

File: _data/nav.yml (hunk 3 above)
Comment (tylerkelly13) [reply _data/nav.yml:265]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1675/commits/9189463c6115d08e4d00e072ea557288f9416116

Leaving APIs, removed command headings from nav (for now).

**Review by EkimChau — APPROVED** [review]

**Review by HAFRMO — APPROVED** [review]

**Pull request merged**